File: relatorio/codigos/ruby/micro03.rb
# le um numero e verifica se ele esta entre 100 e 200

puts("Digite um numero:")
numero = gets.chomp.to_i

if numero >= 100
  if numero <= 200
    puts("O numero esta no intervalo entre 100 e 200")
  else
    puts("O numero nao esta no intervalo entre 100 e 200")
  end
else
  puts("O numero nao esta no intervalo entre 100 e 200")
end

File: relatorio/codigos/lua/micro03.lua
-- Le um numero e verifica se ele esta entre 100 e 200
--[[ Funcao: Faca um algoritmo que receba um numero e diga se este numero esta no intervalo entre 100 e 200 --]]

print("Digite um número:")
numero = io.read("*number")

if(numero >= 100)
then
	if(numero <= 200)
	then
		print("O número está no intervalo entre 100 e 200")
	else
		print("O número não está no intervalo entre 100 e 200")
	end
else
	print("O número não está no intervalo entre 100 e 200")
end